packages/dokeysto_lz4/dokeysto_lz4.3.0.2/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "dokeysto_lz4"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/dokeysto"
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test_lz4"] {with-test}
```

Comment:
```suggestion
  ["dune" "exec" "-p" name "-j" jobs "src/test_lz4.exe"] {with-test}
```
The make command hides a call to dune that doesn't have `-p` or `--profile`. This should not be used in releases

Reply:
The problem is that the Makefile's test target for that does more than just running the test exe.
```
test_lz4:
        \rm -f rwdb_lz4 rwdb_lz4.idx
        dune build _build/default/src/test_lz4.exe
        _build/default/src/test_lz4.exe
```
I'll comment that test line for the moment.

Reply:
I don't see what it does more than just removing file that are not present in the repository,  building and running the tests tbh. All of that are done by `dune exec`

Reply:
ok, I'll put back your test line